scripts/src/main/resources/scripts/s2-repository-activation.js

```diff
@@ -0,0 +1,181 @@
+	    		"\t\t<id>mrm.apps2.capgemini.com</id>\n" +
+        		"\t\t<username>"+user+"</username>\n" + 
+        		"\t\t<password>"+pass+"</password>\n" +
+			"\t</server>";
+}
+
+function injectS2profile(engagementName){
+	return "\t<profile>\n" +
+    			"\t\t<id>s2</id>\n" +
+    			"\t\t<repositories>\n" +  
+      				"\t\t\t<repository>\n" +
+        				"\t\t\t\t<id>mrm.apps2.capgemini.com</id>\n" +
+        				"\t\t\t\t<url>http://mrm.apps2.capgemini.com/"+engagementName+"</url>\n" +
+      				"\t\t\t</repository>\n" +
+    			"\t\t</repositories>\n" +
+			"\t</profile>";
+}
+
+function injectS2activeProfile(){
+	return "\t<activeProfile>s2</activeProfile>";
+}
+
+function injectCIAASservers(user, pass){
+	return "\t<server>\n" +
+      			"\t\t<username>"+user+"</username>\n" +
+      			"\t\t<password>"+pass+"</password>\n" +
+      			"\t\t<id>central</id>\n" +
+    		"\t</server>\n" +
+    		"\t<server>\n" +
+      			"\t\t<username>"+user+"</username>\n" +
+      			"\t\t<password>"+pass+"</password>\n" +
+      			"\t\t<id>snapshots</id>\n" +
+    		"\t</server>";
+}
+
+function injectCIAASprofiles(engagementName){
+	return "\t<profile>\n" +
+      			"\t\t<repositories>\n" +
+        			"\t\t\t<repository>\n" +
+          				"\t\t\t\t<snapshots>\n" +
+            				"\t\t\t\t\t<enabled>false</enabled>\n" +
+          				"\t\t\t\t</snapshots>\n" +
+          				"\t\t\t\t<id>central</id>\n" +
+          				"\t\t\t\t<name>"+engagementName+"</name>\n" +
+          				"\t\t\t\t<url>http://mrm.apps2.capgemini.com/"+engagementName+"</url>\n" +
+        			"\t\t\t</repository>\n" +
+        			"\t\t\t<repository>\n" +
+          				"\t\t\t\t<snapshots />\n" +
+          				"\t\t\t\t<id>snapshots</id>\n" +
+          				"\t\t\t\t<name>"+engagementName+"</name>\n" +
+          				"\t\t\t\t<url>http://mrm.apps2.capgemini.com/"+engagementName+"</url>\n" +
+        			"\t\t\t</repository>\n" +
+      			"\t\t</repositories>\n" +
+      			"\t\t<pluginRepositories>\n" +
+        			"\t\t\t<pluginRepository>\n" +
+          				"\t\t\t\t<snapshots>\n" +
+            				"\t\t\t\t\t<enabled>false</enabled>\n" +
+          				"\t\t\t\t</snapshots>\n" +
+          				"\t\t\t\t<id>central</id>\n" +
+          				"\t\t\t\t<name>"+engagementName+"</name>\n" +
+          				"\t\t\t\t<url>http://mrm.apps2.capgemini.com/"+engagementName+"</url>\n" +
+        			"\t\t\t</pluginRepository>\n" +
+        			"\t\t\t<pluginRepository>\n" +
+          				"\t\t\t\t<snapshots />\n" +
+          				"\t\t\t\t<id>snapshots</id>\n" +
+          				"\t\t\t\t<name>"+engagementName+"</name>\n" +
+          				"\t\t\t\t<url>http://mrm.apps2.capgemini.com/"+engagementName+"</url>\n" +
+        			"\t\t\t</pluginRepository>\n" +
+      			"\t\t</pluginRepositories>\n" +
+      			"\t\t<id>artifactory</id>\n" +
+    		"\t</profile>";
+}
+
+function injectCIAASactiveProfile(){
+	return "\t<activeProfile>artifactory</activeProfile>";
+}
```

Comment:
I still do not understand what was the real purpose behind all this "s2" stuff.
It seems somewhat more flexible to generate the settings.xml dynamically but putting in passwords this way is not best practice to me.
http://maven.apache.org/guides/mini/guide-encryption.html